org.eclipse.winery.repository.ui/src/index.html

```diff
@@ -14,17 +14,11 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <!--
-    The base MUST be set before ANYTHING else is referenced, otherwise angular won't find anything.
-    If hosted on a tomcat server, the base MUST be href="./".
-    If hosted on lite server, it MUST be href="/".
-    -->
-    <base href="/">
+    <base href="./">
```

Comment:
No, angular needs the base reference for the navigation, see [Angular Router](https://angular.io/guide/router#base-href).